### DISCUSSION TITLE
is utils.losses redundant?

### DISCUSSION THREAD

**Discussion by Philip1435** [qubvel/segmentation_models.pytorch#887] [General]
I was working with Diceloss and found that there are two places it is written: `losses/dice` and `utils/losses`. I used the latter one at first and did not get the outcome I expected, but then found success with the first one. I am here asking if there is any use for `utils/losses` or is it actually redundant and someone forgot to clean it?

**Comment by qubvel** [comment]
Hi, thanks for the question, `utils` is deprecated and going to be removed
